hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/webapp/MockDefaultRequestInterceptorREST.java

```diff
@@ -412,4 +419,50 @@ public Response signalToContainer(String containerId, String command,
 
     return Response.status(Status.OK).build();
   }
+
+  @Override
+  public org.apache.hadoop.yarn.server.webapp.dao.AppAttemptInfo getAppAttempt(HttpServletRequest req, HttpServletResponse res,
```

Comment:
We need to specify the whole type? Where is the conflict?

Reply:
I found 2 conflicts

org.apache.hadoop.yarn.server.resourcemanager.webapp.dao.AppAttemptInfo
org.apache.hadoop.yarn.server.webapp.dao.AppAttemptInfo

MockDefaultRequestInterceptorREST#getAppAttempts, I will use whole type.
```
public AppAttemptsInfo getAppAttempts(HttpServletRequest hsr, String appId) {
    if (!isRunning) {
      throw new RuntimeException("RM is stopped");
    }

    ApplicationId applicationId = ApplicationId.fromString(appId);
    if (!applicationMap.contains(applicationId)) {
      throw new NotFoundException("app with id: " + appId + " not found");
    }

    AppAttemptsInfo infos = new AppAttemptsInfo();
    org.apache.hadoop.yarn.server.resourcemanager.webapp.dao.AppAttemptInfo
         appAttemptInfo1 = new org.apache.hadoop.yarn.server.resourcemanager.webapp.dao.AppAttemptInfo();
    infos.add(appAttemptInfo1);

    return infos;
  }
```